client/components/form/Radio.tsx

```diff
@@ -18,10 +19,11 @@ export interface RadioType extends React.FC<RadioProps> {
   Boolean: React.FC<BooleanRadioProps>;
 }
 
-export const Radio: RadioType = ({ legend, name, options, horizontal }) => {
+export const Radio: RadioType = ({ title, legend, name, options, horizontal }) => {
   return (
-    <fieldset className='flex flex-col gap-4'>
-      <legend className='text-bcBlack font-bold mb-4'>{legend}</legend>
+    <div className='flex flex-col gap-2'>
+      <h3 className='text-bcBlack font-bold w-full'>{title}</h3>
```

Comment:
See comment [here](https://github.com/bcgov/PBGP/pull/15#discussion_r1005014374) about possible accessibility issues here

In this case, title won't be properly exposed to assistive technology as it isn't present in the legend, passing in styled markup is probably the best solution here

Reply:
Altered the div back to fieldset and legend with a p tag for description. This should solve the accessibility issues.